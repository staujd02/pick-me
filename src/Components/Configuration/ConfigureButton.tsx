import React from 'react';

export default class ConfigureButton extends React.Component<ConfigureButtonProps> {
    render(){
        const { buttonClickHandler } = this.props;
        return <button onClick={buttonClickHandler}>Configure Streams</button>
    }
}