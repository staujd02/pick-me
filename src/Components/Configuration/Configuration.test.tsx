import React from 'react';
import { formatHTML } from '../../TestUtilities/htmlFormatter';
import { shallow, ShallowWrapper } from 'enzyme';
import Configuration from './Configuration';

describe('The Configuration Component', () => {

    let wrapper: ShallowWrapper<{},{},Configuration>

    beforeEach(() => {
        wrapper = shallow(<Configuration />);
    });

    it('renders correctly', 
        () => expect(formatHTML(wrapper.html())).toMatchSnapshot());

});