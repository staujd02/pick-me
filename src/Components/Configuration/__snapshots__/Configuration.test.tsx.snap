// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Configuration Component renders correctly 1`] = `
"
<div></div>
"
`;
